tests/SchemeSpec.hs

```diff
@@ -52,25 +49,29 @@ someED25519Pair = (PubBS $ getByteString
                    Nothing,
                    ED25519)
 
+anotherED25519Pair :: (PublicKeyBS, PrivateKeyBS, Address, PPKScheme)
+anotherED25519Pair = (PubBS $ getByteString
+                      "6866b33e7935752bb972f363fe0567902616075878392ff7159f5fd4a2672827",
+                      PrivBS $ getByteString
+                      "7693e641ae2bbe9ea802c736f42027b03f86afe63cae315e7169c9c496c17331",
+                      Just "6866b33e7935752bb972f363fe0567902616075878392ff7159f5fd4a2672827",
+                      ED25519)
 
-
-
-someETHPair :: (PublicKeyBS, PrivateKeyBS, Address, PPKScheme)
-someETHPair = (PubBS $ getByteString
-               "836b35a026743e823a90a0ee3b91bf615c6a757e2b60b9e1dc1826fd0dd16106f7bc1e8179f665015f43c6c81f39062fc2086ed849625c06e04697698b21855e",
-               PrivBS $ getByteString
-               "208065a247edbe5df4d86fbdc0171303f23a76961be9f6013850dd2bdc759bbb",
-               Just $ "0bed7abd61247635c1973eb38474a2516ed1d884",
-               ETH)
-
+someWebAuthnSignature :: (UserSig, PublicKeyBS)
+someWebAuthnSignature = (sig, pubKey)
```

Comment:
This seems to be our only coverage. Are there any code paths this does not hit that aren't just a fatal parsing problem?

Reply:
:+1: I've added tests for cmdHash mismatch, signature scheme not set to WebAuthn, and public key mismatch (all three cause verification to fail)